README.md

```diff
@@ -135,6 +136,90 @@ features are used, you can ignore those messages as these features do not affect
+2017. The supported compiler is Microsoft Visual C++ 2015, not Microsoft Visual C++ 2017.**
+
+1. Download and install the Microsoft Visual C++ 2015 compiler (typically achieved by installing
+[Microsoft Visual Studio 2017 Community Edition](https://www.visualstudio.com/downloads)).
+2. Download and install the [Codeplay ComputeCpp Community Edition SDK for Windows]().
+3. Open either _VS2015 x64 Native Tools Command Prompt_ or _Developer Command Prompt for VS 2017_.
+4. Enter the following commands
+```bash
+mkdir SyclParallelSTL
+cd SyclParallelSTL
+mkdir build-debug
+git clone https://github.com/KhronosGroup/SyclParallelSTL.git
+cd build-debug
+cmake -G"Visual Studio 14 2015 Win64" -DCOMPUTECPP_PACKAGE_ROOT_DIR=<path-to-ComputeCpp> ../SyclParallelSTL
+```
+
+where `<path-to-ComputeCpp>` is the path to your ComputeCpp directory. For example, a default
+installation of ComputeCpp on a 64-bit build of Windows 10 will be `C:\\Program Files\\Codeplay\\ComputeCpp`.
+
+5. Open `SyclSTL.sln` using Microsoft Visual Studio. If you are not using Visual Studio 2015, be 
+   sure not to convert the solution to a later version.
+6. Right-mouse click on `ALL_BUILD (Visual Studio 2015)` and select `Build`.
+
+### Microsoft Windows Subsystem for Linux (Bash on Windows)
+
+#### Prerequisites
+
+* Ensure that WSL has been installed after installing the Windows 10 Fall Creators Update (version
+  1709). You can do this by opening Bash on Windows and typing the command `lsb_release -a` and
+* CMake 3.5 or later (either installed natively or built from source)
+* GCC 5 or later
+
+#### Installing Windows Subsystem for Linux (a.k.a. Bash for Windows)
+
+1. Ensure that you have installed Windows 10 Fall Creators Update (version 1709), or later.
+2. Open Start Menu
+3. Type 'Developers Settings' and press Enter
+4. Under 'Use developer features', ensure that 'Developer mode' is selected.
+5. If prompted to reboot your computer, do not reboot.
+6. Open Computer
+7. Under the Computer tab, select 'Uninstall or change a program'
+8. In the left-hand pane, select 'Turn Windows features on or off'
+9. Scroll to the very bottom and select 'Windows Subsystem for Linux (Beta)'
+10. Reboot Windows
+11. Open Start Menu
+12. Type 'Bash' and press Enter. Make sure that you aren't opening a mintty client such as Git Bash or Cygwin.
+13. A command prompt window should open, and take you through the final part of the installation process.
+
+#### Install developer tools
+
+1. Type the following commands into Bash. Replace `<username>` with your Windows username.
+
+```bash
+sudo apt update && sudo apt upgrade && sudo apt update
+sudo apt install build-essentials binutils gdb git flex bison texlive-full
+mkdir /mnt/c/Users/<username>/projects; cd /mnt/c/Users/<username>/projects
+git clone https://github.com/Kitware/CMake.git
+cd CMake
+./bootstrap && make -j 4 && sudo make install
+cd ..
+sudo apt install ocl-icd-libopencl1 ocl-icd-dev opencl-headers clinfo lsb-core
```

Comment:
clinfo is not required as the user can run computecpp_info to get the OpenCL devices. this is also more relevant as the user can check which devices work with computeCpp